_markbind/navigation/mainNav.md

```diff
@@ -65,10 +65,10 @@
   * [Android]({{baseUrl}}/contents/android/IntroToAndroid.html)
   * [Angular]({{baseUrl}}/contents/javascript/Javascript-framework-Angular.html)
   * [iOS]({{baseUrl}}/contents/ios/IntroToIos.html)
-  * [NodeJS]({{baseUrl}}/contents/node/nodejs.html)
+  * [Node.js]({{baseUrl}}/contents/node/nodejs.html)
   * [React]({{baseUrl}}/contents/javascript/Javascript-framework-React.html)
```

Comment:
Should I ake node.js Node for standardisation? @yash-chowdhary

Reply:
I don't mind, but that would mean making use of "Node" throughout.

Reply:
Cool